src/ansys/aedt/toolkits/antenna/ui/frontend_properties.toml

```
@@ -0,0 +1,61 @@
+[defaults]
+toolkit_name= "antenna"
+backend_url= "127.0.0.1"
+backend_port= 5001
+debug= true
+app_name= "Antenna Wizard"
+main_title= "Antenna Toolkit"
+welcome_message= "Welcome to the antenna toolkit"
```

```suggestion
main_title= "Antenna Toolkit"
welcome_message= "Welcome to the antenna toolkit"
```
```suggestion
main_title= "AEDT Antenna Toolkit"
welcome_message= "Welcome to the AEDT Antenna Toolkit"
```
Use consistent proudct name. Where does this info get shown. For copyright info, its always ANSYS, Inc.